src/components/chat-view-container/chat-view.tsx

```diff
@@ -240,10 +243,28 @@ export class ChatView extends React.Component<Properties, State> {
             )}
             {this.props.messages.length > 0 && <Waypoint onEnter={this.props.onFetchMore} />}
             {this.props.messages.length > 0 && this.renderMessages()}
-            {!this.props.hasLoadedMessages && <ChatSkeleton conversationId={this.props.id} />}
+            {this.props.messagesFetchStatus === MessagesFetchState.IN_PROGRESS && (
+              <ChatSkeleton conversationId={this.props.id} />
+            )}
+
             <div ref={this.bottomRef} />
           </div>
         </InvertedScroll>
+
+        {this.props.messagesFetchStatus === MessagesFetchState.FAILED && (
+          <div className='channel-view__failure-message'>
+            Failed to load messages.&nbsp;
```

Comment:
The message is not per design (`Failed to load messages`), and will change. I have asked eth/kelly for a message to show when you're in a group conversation or a normal zOS channel